playbooks/roles/generate-certs/handlers/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+# handlers file for generate-certs
```

Comment:
Is this currently necessary?

Reply:
It's generated by default by ansible-galaxy :p 
I don't mind removing it. I don't see a point either.